backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -716,6 +721,40 @@ def test_application_date_range(
         assert response.data["end_date"] == end_date
 
 
+@pytest.mark.parametrize(
```

Comment:
Maybe freezing the time for this test would be good, e.g.
`@freeze_time("2021-01-01")`?

Reply:
freeze_time is applied to all these tests automatically (from common.tests.conftest)

Reply:
Ah, ok so it was `from common.tests.conftest import *` in `backend/benefit/applications/tests/test_applications_api.py` which imported the autouse fixture `setup_test_environment` from `common.tests.conftest` which made the freeze_time be applied to these tests. Alright!